src/assets/app.scss

```diff
@@ -0,0 +1,26 @@
+@import 'custom';
```

Comment:
I would suggest we have a sub folder under the assets named styles. so that we can keep other assets like icons and images in their appropriate folder.